spec/models/manageiq/providers/embedded_ansible/provider_spec.rb

```diff
@@ -2,4 +2,44 @@
 
 describe ManageIQ::Providers::EmbeddedAnsible::Provider do
   it_behaves_like 'ansible provider'
+
+  subject { FactoryGirl.build(:provider_embedded_ansible) }
```

Comment:
From https://github.com/ManageIQ/manageiq/pull/14283#discussion_r106753956 
>@bdunne
>Why not FactoryGirl.create rather than adding the subject.save in the before block?

>@carbonin
>Ah, I think this was a rebase issue. I started this branch before #14383 was merged so instead of changing the subject for all the tests, I saved the record in just my context, during the rebase I think this line got left here.

>I'll refactor in a new commit so that the individual specs define the subject rather than the shared example.

>Thanks for catching this.